third_party/nvfuser/csrc/fusion_segmenter.cpp

```diff
@@ -3231,6 +3232,14 @@ std::string toString(const SegmentCandidateFinderOptions& segment_options) {
   return ss.str();
 }
 
+void MultiGroupFusion::print() {
+  std::cout << "MultiGroupFusion {\n";
+  for (auto& group_ptr : groups_) {
+    detailGroupPrint(std::cout, group_ptr.get());
+  }
+  std::cout << "} //MultiGroupFusion \n";
+}
```

Comment:
Let's stick to `toString`. Fusion should likely be changed too at some point.

Reply:
Did I remove something needed. This is marked as outdated so seems I deleted it either because it wasn't or because I didn't realize it was. Please check @samnordmann

Reply:
it's ok